### PR TITLE
Merge development into master for v1.0.1

### PR DESCRIPTION
**Pull request by ClotildeToullec**

### CODE REVIEW

File: src/BaselineOfFamix/BaselineOfFamix.class.st [hunk 1]
```diff
@@ -108,7 +109,7 @@ BaselineOfFamix >> baseline: spec [
 				group: 'BasicTraits' with: #('Famix-Traits');
 				group: 'EntitiesCompatibility' with: #('Famix-Compatibility-Entities');
 				group: 'ModelCompatibility' with: #('Famix-Compatibility-Generator' 'EntitiesCompatibility');
-				group: 'EntitiesJava' with: #('Famix-Java-Entities');
+				group: 'EntitiesJava' with: #('Famix-Java-Entities' 'Famix-Java-Visitor');
```
Comment (badetitou) [inline src/BaselineOfFamix/BaselineOfFamix.class.st:112]: Is it the correct way to add the Visitor. Maybe it should be in another group (so one can load the model without the visitor)

**Review by badetitou — COMMENTED** [review]
IMO, the modification of modifiers is correct, but, we removed the `#modfiers` keyword... So, we cannot load correctly old generated model.
It would be nice to add a method `modifiers` on the entities (attribute, method, etc.) that return the list of modifiers (as before). We can see `#modifiers` as a  derived property

(Note: my projects have bugs since this modification and I had to implement hot fix...)

File: src/BaselineOfFamix/BaselineOfFamix.class.st (hunk 1 above)
Comment (ClotildeToullec) [reply src/BaselineOfFamix/BaselineOfFamix.class.st:112]: Something like:
```smalltalk
group: 'EntitiesJava' with: #('Famix-Java-Entities');
group: 'EntitiesJavaWithVisitor' with: #('EntitiesJava' 'Famix-Java-Visitor');
```
We could also add 
```smalltalk
group: 'ModelJavaWithVisitor' with: #('Famix-Java-Generator' 'EntitiesJavaWithVisitor');
```

What do you think ?

Comment (badetitou) [reply src/BaselineOfFamix/BaselineOfFamix.class.st:112]: I like the first one
No need of the last one IMO

**Review by badetitou — APPROVED** [review]
It is ok for me.
We know that there will have a little bug with modifiers keyword. But compared to all the new features I believe it is more important to release than fixing this.

Still, we should create a deprecated method for next patch as soon as possible

**Pull request merged**